### ISSUE TITLE
Request: Support TypeScript scanning

### ISSUE THREAD

**Issue by blalasaadri** [zeke/standard-markdown#33]
StandardJS supports scanning TypeScript files as well (as explained [in the FAQs](https://standardjs.com/#can-i-use-a-javascript-language-variant-like-flow-or-typescript)). standard-markdown however only scans for `javascript` or `js` code blocks (as becomes clear from [this line of code](https://github.com/zeke/standard-markdown/blob/master/lib/codeBlockUtils.js#L3) in codeBlockUtils.js).

It would be great to also scan fenced typescript code blocks using the `typescript` or `ts` language identifier (which are both [supported by Linguist](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5189), which is what GitHub flavoured markdown uses).

**Comment by zeke** [comment]
Good idea! I don't have time to implement this, but I would definitely be open to a pull request that adds it.
